massgov/mayflower
# Patternlab/DP-9693 modify body text color to use a darker grey

## Description
Switch body copy color to #141414 ($c-font-base: $c-black) while keeping requested elements #535353 ($c-font-detail: $c-theme-gray)

## Related Issue / Ticket

- [JIRA issue DP-9693](https://jira.mass.gov/browse/DP-9693)


## Steps to Test

1. Per https://jira.mass.gov/browse/DP-9693 open the document found here: https://docs.google.com/document/d/1DInGiHDJX945afVmS6m6czGD0oAJGJgBMI3p0vxjySc/edit

2. Check that body copy is now defined as  #141414 throughout, while elements details in the doc remain #535353

Backstop files are attached
